angular.json

```diff
@@ -28,7 +28,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.scss"
```

Comment:
This was good to learn, as changing from plain css to scss and sass might be important on the job.